### DISCUSSION TITLE
[Question]: What's the difference between the Toolkit and the Extended Toolkit?

### DISCUSSION THREAD

**Discussion by lynnewu** [Krypton-Suite/Extended-Toolkit#333] [Q&A] [answered]
What's the difference between the Standard Toolkit and the Extended Toolkit?

**Comment by Smurf-IV** [comment]
The Idea of the standard toolkit, is to give the "Standard" winform controls theming, with a few extras to help with the controls data.
It also has ribbons / Docking / navigation components that came form the initial implementation before they became the norm in Winforms. 

Extended, then takes those controls and adds new features / new controls / new anything that gives the developer more control over data / interactions / display, e.g. the "ProgressBar with text" is one of the recent additions into Extended

**Reply by lynnewu** [reply]
Thanks!

Do I need to reference "Standard" if I'm using "Extended", then?

**Comment by mikel3361** [comment, marked as answer]
Extended NuGet Packages refernces Standard Packages when needed and install it. I'm only install Extended and the standard toolkit is also referenced.

**Comment by Smurf-IV** [comment]
That's because the Extended Items "Derive from and / or Extend" the Toolkit, and also need all the palette and theming that krypton does in the standard toolkit.
